.circleci/config.yml

```diff
@@ -42,16 +42,34 @@ jobs:
             - ~/workbench/api/build/exploded-api/WEB-INF/lib/
           key: api-cache-{{ checksum "~/workbench/api/build.gradle" }}
       - deploy:
-          name: Deploy to gcloud test project
+          name: Maybe deploy to test/staging
           working_directory: ~/workbench/api
           command: |
-            ./project.rb circle-deploy \
-              --project all-of-us-workbench-test --creds-file circle-sa-key.json
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              ./project.rb deploy \
+                --project all-of-us-workbench-test \
+                --account=circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com \
+                --version circle-ci-test \
+                --promote
+            elif [ -n "${CIRCLE_TAG}" ]; then
```

Comment:
Regarding API versioning convention, I'm not sure we have one yet, but we should make one. We should probably use the same version for api and UI. Maybe workbench_vX_Y_rcZ?

We probably *don't* want to autodeploy to staging on every tag created. Which I think means we need some kind of check.

Reply:
I'm not sure I would expect people to be creating random tags, as that means you needed to either click "new release" via the UI, or explicitly push your own local tags up, I believe.

That said, we can have a convention here. I don't see any need for a prefix here, my own preference would be for standard semantic versioning (with a "v" prefix as a few places recommend, to avoid leading with a number), i.e. `v1.0.0`. I'm also OK just copying the current RDR pattern of `v1-0-rc1` for consistency.
 
Switched this to pattern match on `v*`

Reply:
OK. We presently create staging candidate for RDR by going to the new releases URL and entering a tag of a particular form. I was assuming we'd do something similar for workbench. Let me know if you have other thoughts.

vX-Y-rcZ sounds good.